packages/backend/src/graphql/schema.graphql

```diff
@@ -163,6 +163,7 @@ input CreateAssetInput {
   scale: UInt8!
   "Minimum amount of liquidity that can be withdrawn from the asset"
   withdrawalThreshold: UInt64
+  idempotencyKey: String!
```

Comment:
for now, just added to `createAsset` mutation, wanted to keep PR as small as possible for initial review